components/gridlayout/overview.md

```diff
@@ -0,0 +1,160 @@
+
+## Basics
+
+#### To use a Telerik GridLayout for Blazor
+
+1. Declare the `<TelerikGridLayout>` tag 
+
+1. To define columns in the GridLayout add one or more instances of the `<GridLayoutColumn>` to the `<GridLayoutColumns>` collection.
+
+    * The number of `<GridLayoutColumn>` should be the same as the number of the `<GridLayoutItem>`.
+    
+1. To define rows in the GridLayout add one or more instances of the `<GridLayoutRow>` to the `<GridLayoutRows>` collection.
+
+1. Provide one or more instances of the `<GridLayoutItem>` to the `<GridLayoutItems>` collection.
+
+>caption A basic configuration of the Telerik GridLayout
+
+````CSHTML
+@* Split the items in the GridLayout in three columns and two rows. *@
+
+<TelerikGridLayout>
+    <GridLayoutColumns>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+    </GridLayoutColumns>
+    <GridLayoutRows>
+        <GridLayoutRow Height="50px"></GridLayoutRow>
+    </GridLayoutRows>
+    <GridLayoutItems>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 1
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 2
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 3
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 4
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 5
+            </div>
+        </GridLayoutItem>
+    </GridLayoutItems>
+</TelerikGridLayout>
+````
+
+>caption The result from the code snippet above
+
+![overview of the GridLayout functionality](images/gridlayout-overview-basic-example.png121)
+
+
+>caption Component namespace and reference
+
+````CSHTML
+<TelerikGridLayout @ref="@GridLayoutRef">
+    <GridLayoutColumns>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+    </GridLayoutColumns>
+    <GridLayoutRows>
+    </GridLayoutRows>
+    <GridLayoutItems>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 1
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 2
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 3
+            </div>
+        </GridLayoutItem>
+    </GridLayoutItems>
+</TelerikGridLayout>
+
+@code {
+    Telerik.Blazor.Components.TelerikGridLayout GridLayoutRef { get; set; }
+}
+````
+
+
+## Features
+
+The StackLayout offers the following features:
```

Comment:
We are introducing a StackLayout term, which has not been defined. Is this a typo? Is this a feature of the GridLayout? Is this a separate component? How does it fit in the above information and snippets?

Reply:
Indeed, it is a typo. Will be fixed with the rest of the improvements based on your feedback